spec/policies/agent/agent_policy_spec.rb

```diff
@@ -85,6 +85,21 @@
       end
     end
 
+    context "agent is in secrétariat" do
+      let!(:service_secretariat) { create(:service, :secretariat) }
+      let!(:other_service) { create :service }
+      let!(:services) { create_list(:service, 2) }
+      let!(:organisations) { create_list(:organisation, 2) }
+      let!(:agent) { create(:agent, basic_role_in_organisations: organisations, service: service_secretariat) }
+      let!(:other_agent_same_orgas) { create(:agent, basic_role_in_organisations: organisations, service: other_service) }
+
+      it do
+        expect(subject).to include(agent)
+        expect(subject).to include(other_agent_same_orgas)
+        expect(subject.count).to eq(2)
```

Comment:
pour vérifier qu'on a bien ces deux agents de manière un peu concise, on peut utiliser le matcher `match_array`, ça permet aussi d'afficher les éléments en trop quand le test échoue. et vu qu'il y a un seul expect, ça permet même d'utiliser `is_expected` pour faire ça en une ligne, du genre ` it { is_expected.to match_array([agent, other_agent_same_orgas]) }`

Reply:
Yep, ça sera plus simple, tu as raison. J’ai un peu copié-collé sans réfléchir.

(Est-ce que ça se voit que j’ai écrit le test après le fix? 😇)